examples/notebooks/armory_colab_example.ipynb

```diff
@@ -0,0 +1,1232 @@
+            "Requirement already satisfied: frozenlist>=1.1.1 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from aiohttp->datasets) (1.3.3)\n",
+            "Requirement already satisfied: attrs>=17.3.0 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from aiohttp->datasets) (23.1.0)\n",
+            "Requirement already satisfied: aiosignal>=1.1.2 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from aiohttp->datasets) (1.3.1)\n",
+            "Requirement already satisfied: multidict<7.0,>=4.5 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from aiohttp->datasets) (6.0.4)\n",
+            "Requirement already satisfied: charset-normalizer<4.0,>=2.0 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from aiohttp->datasets) (3.2.0)\n",
+            "Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from aiohttp->datasets) (4.0.2)\n",
+            "Requirement already satisfied: typing-extensions>=3.7.4.3 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from huggingface-hub<1.0.0,>=0.14.0->datasets) (4.5.0)\n",
+            "Requirement already satisfied: filelock in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from huggingface-hub<1.0.0,>=0.14.0->datasets) (3.12.2)\n",
+            "Requirement already satisfied: certifi>=2017.4.17 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from requests>=2.19.0->datasets) (2022.12.7)\n",
+            "Requirement already satisfied: urllib3<3,>=1.21.1 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from requests>=2.19.0->datasets) (1.26.16)\n",
+            "Requirement already satisfied: idna<4,>=2.5 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from requests>=2.19.0->datasets) (2.10)\n",
+            "Requirement already satisfied: python-dateutil>=2.8.2 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from pandas->datasets) (2.8.2)\n",
+            "Requirement already satisfied: tzdata>=2022.1 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from pandas->datasets) (2023.3)\n",
+            "Requirement already satisfied: pytz>=2020.1 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from pandas->datasets) (2023.3)\n",
+            "Requirement already satisfied: six>=1.5 in /home/chris/.virtualenvs/cdao/lib/python3.10/site-packages (from python-dateutil>=2.8.2->pandas->datasets) (1.16.0)\n"
+          ]
+        }
+      ],
+      "source": [
+        "!pip install armory-library\n",
+        "!pip install datasets"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "BVyMdIKHzIQq"
+      },
+      "source": [
+        "### Imports for example\n",
+        "### These are all the necessary imports to run the example file and will all be used and explained later in the notebook"
```

Comment:
I don't know how this looks in colab, is this being rendered as a level 3 header?

Reply:
Yes it is. It was too large in my opinion when it was level 2 or 3. Here is a screenshot for reference. If you don't agree I can change?

![image](https://github.com/twosixlabs/armory-library/assets/35696217/f96f1e9c-a434-4ceb-b86a-d1d757ad619d)

Reply:
It looks fine to me. That's the downside to only being able to comment on the raw notebook source. I wouldn't have expected the "These are all..." sentence to be a header at all.